tests/lib/snaps/content-slot/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: content-slot
+version: 1.0
+slots:
+    shared-content-slot:
+        interface: content
+        content: mylib
+        read:
+            - $SNAP_DATA
```

Comment:
I'm not sure this works, could you please try just `/` here? The path is always relative to $SNAP_DATA here.